### PR TITLE
Localisation Crusade 19

### PR DESCRIPTION
**Pull request by arithon**
## Changelog:
- Deleted RL astrology connection.
- Fixed MNM.1600.
- Fixed horse relation in HL.4814 event text.
- Fixed "at peace for too long" description.
- Translation and fixes to hermetic lines.
- Added some more Xs in localisation files to close the line.
- Added missing religion description translations.
- Fixed some minor stuff in religious csv.
- Some more french localisation.
- German generic and random artifact loc improved, origins translated (but might need some more custom localisation definitions)

<!--

### CODE REVIEW

File: localisation/00_wc_artifact_localisation.csv [hunk 1]
```diff
@@ -105,21 +105,15 @@ whispers_learning_4_desc;When someone holds this book, dark whispers creep into
...
+flute_of_accuracy;Flute of Accuracy;Fl�te de pr�cision d'Alleria;Fl�te der Genauigkeit;;;;;;;;;;;x
+flute_of_accuracy_desc;A wooden flute set in gold. It is unclear whether the flutes of accuracy were named in honour of the elven ranger Alleria or whether she was involved in her invention.;Une fl�te en bois sertie d'or. On ne sait pas si les fl�tes de pr�cision ont �t� nomm�es en l'honneur de la foresti�re elfe Alleria ou si elle a particip� � son invention.;Eine in Gold gefasste h�lzerne Fl�te. Es ist unklar, ob die Fl�ten der Genauigkeit zu Ehren der elfischen Waldl�uferin Alleria benannt wurden oder ob sie an ihrer Erfindung beteiligt war.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_wc_artifact_localisation.csv:111]: Changed her to their
```suggestion
flute_of_accuracy_desc;A wooden flute set in gold. It is unclear whether the flutes of accuracy were named in honour of the elven ranger Alleria or whether she was involved in their invention.;Une fl�te en bois sertie d'or. On ne sait pas si les fl�tes de pr�cision ont �t� nomm�es en l'honneur de la foresti�re elfe Alleria ou si elle a particip� � son invention.;Eine in Gold gefasste h�lzerne Fl�te. Es ist unklar, ob die Fl�ten der Genauigkeit zu Ehren der elfischen Waldl�uferin Alleria benannt wurden oder ob sie an ihrer Erfindung beteiligt war.;;;;;;;;;;;x
```

File: localisation/00_wc_sad.csv [hunk 2]
```diff
@@ -730,24 +730,25 @@ EVTOPTA_WCUTF.3;Forgive me, my Makers.;Pardonnez-moi, mes Cr
...
-GetOldGodLairNameOnWonder_Ulduar;Ulduar;;;;;;;;;;;;;x
-GetOldGodLairDescOnWonder_AhnQiraj;the Old God's lair, the dangerous and fortified city of Ahn'Qiraj in the southern desert of Silithus;;;;;;;;;;;;;x
-GetOldGodLairDescOnWonder_Nyalotha;the Old God's realm, the dangerous and maddening city of Ny'alotha;;;;;;;;;;;;;x
-GetOldGodLairDescOnWonder_Ulduar;the Old God's lair, the dangerous and fortified titan complex of Ulduar in the northern mountains of Storm Peaks;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;
+create_blood_knights_order;Create Blood Knights Order;Cr�er l'Ordre des Chevaliers de sang;Blutritter-Orden schaffen;;;;;;;;;;;x
+create_blood_knights_order_desc;A long time ago, the elves suffered and have been threatened by enemies. Now, this order of elven paladins will help all of elfkind.;Il y a longtemps, les Elfes ont souffert et ont �t� menac�s par des ennemis. Aujourd'hui, cet ordre de paladins elfiques va aider l'ensemble du peuple elfe.;Vor langer Zeit haben die Elfen sehr unter der Bedrohung durch Feinde gelitten und wurden von ihnen bedroht. Jetzt wird dieser Orden der elfischen Paladine die gesamte Elfenwelt sch�tzen.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_wc_sad.csv:735]: Changed have been to were
```suggestion
create_blood_knights_order_desc;A long time ago, the elves suffered and were threatened by enemies. Now, this order of elven paladins will help all of elfkind.;Il y a longtemps, les Elfes ont souffert et ont �t� menac�s par des ennemis. Aujourd'hui, cet ordre de paladins elfiques va aider l'ensemble du peuple elfe.;Vor langer Zeit haben die Elfen sehr unter der Bedrohung durch Feinde gelitten und wurden von ihnen bedroht. Jetzt wird dieser Orden der elfischen Paladine die gesamte Elfenwelt sch�tzen.;;;;;;;;;;;x
```

File: localisation/00_wc_artifact_localisation.csv [hunk 3]
```diff
@@ -105,21 +105,15 @@ whispers_learning_4_desc;When someone holds this book, dark whispers creep into
 deathwhisper;Deathwhisper;Murmemort;Deathwhisper;;;;;;;;;;;x
 deathwhisper_desc;After regaining her physical form, the Dark Lady seized this bow infused with powerful dark magic.;Apr�s avoir retrouv� sa forme physique, la Dame noire s'est empar�e de cet arc impr�gn� de puissante magie noire.;Nach ihrem Untod und der Wiedererlangung ihrer Gestalt ergriff die dunkle Lady diesen von m�chtiger dunkler Magie durchdrungenen Bogen.;;;;;;;;;;;x
 # Rare #;;;;;;;;;;;;;;
-kama_sutra;A Steamy Romance Novel;Un roman d'amour torride;Ein schl�pfriger Romantikschm�ker.;;Novela Romantica Torrida;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_wc_artifact_localisation.csv:108]: REE! Moving lines!

File: localisation/00_wc_artifact_localisation.csv (hunk 1 above)
Comment (arithon) [reply localisation/00_wc_artifact_localisation.csv:111]: done

File: localisation/00_wc_sad.csv (hunk 2 above)
Comment (arithon) [reply localisation/00_wc_sad.csv:735]: done

File: localisation/wc_religions.csv [hunk 4]
```diff
@@ -36,60 +36,60 @@ ILLIDARI_TEACHING;Illidari Teachings;Enseignements illidari;Illidans Lehren;;;;;
...
+rationalism_DESC;Rationalism is much of a philosophy for gnomes that are focused on forward-thinking, inventive concepts, and little thought has ever been given to history or non-scientific records. Gnomes place their loyalty in themselves, in their friends, and in their inventions.;Le rationalisme est en grande partie une philosophie pour les Gnomes qui se sont concentr�s sur des concepts avant-gardistes et inventifs. Ils ont peu pens� � l'histoire ou aux documents non scientifiques. Les gnomes placent leur loyaut� en eux-m�mes, en leurs amis et dans leurs inventions.;Der Rationalismus ist eine Philosophie f�r Gnome, die sich auf vorausschauende, erfinderische Konzepte konzentrieren, und �ber Geschichte oder nichtwissenschaftliche Aufzeichnungen wurde noch nie nachgedacht. Gnome setzen ihre Loyalit�t in sich selbst, in ihre Freunde und in ihre Erfindungen.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/wc_religions.csv:65]: Changed much to more.
```suggestion
rationalism_DESC;Rationalism is more of a philosophy for gnomes that are focused on forward-thinking, inventive concepts, and little thought has ever been given to history or non-scientific records. Gnomes place their loyalty in themselves, in their friends, and in their inventions.;Le rationalisme est en grande partie une philosophie pour les Gnomes qui se sont concentr�s sur des concepts avant-gardistes et inventifs. Ils ont peu pens� � l'histoire ou aux documents non scientifiques. Les gnomes placent leur loyaut� en eux-m�mes, en leurs amis et dans leurs inventions.;Der Rationalismus ist eine Philosophie f�r Gnome, die sich auf vorausschauende, erfinderische Konzepte konzentrieren, und �ber Geschichte oder nichtwissenschaftliche Aufzeichnungen wurde noch nie nachgedacht. Gnome setzen ihre Loyalit�t in sich selbst, in ihre Freunde und in ihre Erfindungen.;;;;;;;;;;;x
```

File: localisation/wc_religions.csv [hunk 5]
```diff
@@ -36,60 +36,60 @@ ILLIDARI_TEACHING;Illidari Teachings;Enseignements illidari;Illidans Lehren;;;;;
...
+naaru_DESC;The light manifests itself as a golden, shining light energy. The naaru, enigmatic beings composed of shards of fractured light taught the use of the light to the draenei and other enlightened races, but others have come into the study and practice of light-based magic on their own accord.;La Lumi�re se manifeste sous la forme d'une �nergie lumineuse dor�e et brillante. Les Naaru, �tres �nigmatiques compos�s d'�clats de lumi�re fractur�s, ont enseign� l'utilisation de la Lumi�re aux Draenei et autres races �clair�es, mais d'autres sont venus de leur propre chef � l'�tude et � la pratique de la magie bas�e sur la Lumi�re.;Das Licht manifestiert sich als eine goldene, leuchtende Lichtenergie. Die Naaru, r�tselhafte Wesen, die aus Scherben gebrochenen Lichts bestehen, lehrten die Draenei und andere erleuchtete Rassen den Gebrauch des Lichts, aber auch andere haben von sich aus das Studium und die Praxis der auf Licht basierenden Magie erlernt.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/wc_religions.csv:89]: Capitalized three cases of the word 'light'
```suggestion
naaru_DESC;The Light manifests itself as a golden, shining light energy. The naaru, enigmatic beings composed of shards of fractured light taught the use of the Light to the draenei and other enlightened races, but others have come into the study and practice of Light-based magic on their own accord.;La Lumi�re se manifeste sous la forme d'une �nergie lumineuse dor�e et brillante. Les Naaru, �tres �nigmatiques compos�s d'�clats de lumi�re fractur�s, ont enseign� l'utilisation de la Lumi�re aux Draenei et autres races �clair�es, mais d'autres sont venus de leur propre chef � l'�tude et � la pratique de la magie bas�e sur la Lumi�re.;Das Licht manifestiert sich als eine goldene, leuchtende Lichtenergie. Die Naaru, r�tselhafte Wesen, die aus Scherben gebrochenen Lichts bestehen, lehrten die Draenei und andere erleuchtete Rassen den Gebrauch des Lichts, aber auch andere haben von sich aus das Studium und die Praxis der auf Licht basierenden Magie erlernt.;;;;;;;;;;;x
```

File: localisation/wc_religions.csv [hunk 6]
```diff
@@ -36,60 +36,60 @@ ILLIDARI_TEACHING;Illidari Teachings;Enseignements illidari;Illidans Lehren;;;;;
...
...ologies that study arcane magic. Order is most commonly perceived in reality as arcane magic. Arcane energy is innately volatile, and wielding it requires intense precision and concentration. Wielders of arcane magic have learned how to exploit that arcane instability being able to control the elements and other forces such as time.;Les religions ou id�ologies qui �tudient la magie des arcanes. L'ordre est le plus souvent per�u en r�alit� comme de la magie des arcanes. L'�nergie des arcanes est intrins�quement volatile, et la manier requiert une pr�cision et une concentration intenses. Les adeptes de la magie des arcanes ont appris � exploiter cette instabilit� des arcanes en �tant capables de contr�ler les �l�ments et d'autres forces comme le temps.;Religionen oder Ideologien, die sich mit arkaner Magie besch�ftigen. Ordnung wird in der Realit�t am h�ufigsten als arkane Magie wahrgenommen. Arkane Energie ist von Natur aus instabil und sie zu handhaben erfordert h�chste Pr�zision und Konzentration. Die Beherrscher der arkanen Magie haben gelernt, diese arkane Instabilit�t auszunutzen, da sie in der Lage sind, die Elemente und andere Kr�fte wie die Zeit zu kontrollieren.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/wc_religions.csv:92]: Added a comma after instability
```suggestion
arcane_religion_DESC;Religions or ideologies that study arcane magic. Order is most commonly perceived in reality as arcane magic. Arcane energy is innately volatile, and wielding it requires intense precision and concentration. Wielders of arcane magic have learned how to exploit that arcane instability, being able to control the elements and other forces such as time.;Les religions ou id�ologies qui �tudient la magie des arcanes. L'ordre est le plus souvent per�u en r�alit� comme de la magie des arcanes. L'�nergie des arcanes est intrins�quement volatile, et la manier requiert une pr�cision et une concentration intenses. Les adeptes de la magie des arcanes ont appris � exploiter cette instabilit� des arcanes en �tant capables de contr�ler les �l�ments et d'autres forces comme le temps.;Religionen oder Ideologien, die sich mit arkaner Magie besch�ftigen. Ordnung wird in der Realit�t am h�ufigsten als arkane Magie wahrgenommen. Arkane Energie ist von Natur aus instabil und sie zu handhaben erfordert h�chste Pr�zision und Konzentration. Die Beherrscher der arkanen Magie haben gelernt, diese arkane Instabilit�t auszunutzen, da sie in der Lage sind, die Elemente und andere Kr�fte wie die Zeit zu kontrollieren.;;;;;;;;;;;x
```

File: localisation/wc_religions.csv [hunk 7]
```diff
@@ -140,9 +140,9 @@ loa_group;Loa;Loa;Loa;;;;;;;;;;;x
...
+old_gods_worship_DESC;The worship of the Old Gods, mysterious, godlike and great malefic horrors. It is a widespread nihilistic religion based on cults that seeks to bring the end of the world and fanatically serves and worships the old gods, parasitic eldritch horrors who are physical manifestations of the Void.;Le culte des Dieux tr�s anciens, de myst�rieuses, divines et grandes horreurs mal�fiques. C'est une religion nihiliste tr�s r�pandue, bas�e sur des cultes, qui cherche � apporter la fin du monde et qui sert et v�n�re fanatiquement les Dieux tr�s anciens, des horreurs parasites qui sont des manifestations physiques du Vide.;Die Anbetung der alten G�tter, geheimnisvoller, gottgleicher und b�sartiger Schrecken. Es handelt sich um eine weit verbreitete nihilistische Religion, die sich auf Kulte st�tzt, die das Ende der Welt herbeif�hren wollen und fanatisch den alten G�ttern dienen und sie verehren, parasit�re und unheimliche Schrecken, die physische Manifestationen der Leere darstellen.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/wc_religions.csv:145]: Capitalized Old Gods
```suggestion
old_gods_worship_DESC;The worship of the Old Gods, mysterious, godlike and great malefic horrors. It is a widespread nihilistic religion based on cults that seeks to bring the end of the world and fanatically serves and worships the Old Gods, parasitic eldritch horrors who are physical manifestations of the Void.;Le culte des Dieux tr�s anciens, de myst�rieuses, divines et grandes horreurs mal�fiques. C'est une religion nihiliste tr�s r�pandue, bas�e sur des cultes, qui cherche � apporter la fin du monde et qui sert et v�n�re fanatiquement les Dieux tr�s anciens, des horreurs parasites qui sont des manifestations physiques du Vide.;Die Anbetung der alten G�tter, geheimnisvoller, gottgleicher und b�sartiger Schrecken. Es handelt sich um eine weit verbreitete nihilistische Religion, die sich auf Kulte st�tzt, die das Ende der Welt herbeif�hren wollen und fanatisch den alten G�ttern dienen und sie verehren, parasit�re und unheimliche Schrecken, die physische Manifestationen der Leere darstellen.;;;;;;;;;;;x
```

File: localisation/wc_religions.csv [hunk 8]
```diff
@@ -151,10 +151,10 @@ voidfarer_DESC;While dangerous, void magic can be carefully used for the benefit
...
+old_ways;Old Way;ancienne voie;Alte Wege;;;;;;;;;;;x
+old_ways_DESC;The old ways are not widespread, but have existed on the fringes of the gilnean society. The religious leaders of what was in gilneas referred to as the "old ways" eventually became "harvest-witches". Those who used their nature powers to augment gilneas' agricultural output during and following its period of industrialisation.;Les anciennes m�thodes ne sont pas tr�s r�pandues, mais elles ont exist� en marge de la soci�t� giln�enne. Les chefs religieux de ce qu'on appelait � Giln�as les � anciennes m�thodes � ont fini par devenir des � sorciers des r�coltes �. Ce sont ceux qui ont utilis� leurs pouvoirs naturels pour augmenter la production agricole de Giln�as pendant et apr�s sa p�riode d'industrialisation.;Die alten Wege sind nicht weit verbreitet, existierten aber am Rande der gilneanischen Gesellschaft. Die religi�sen F�hrer dessen, was in Gilneas als die �alten Wege� bezeichnet wurde, wurden schlie�lich zu �Erntehexen�, die ihre Naturkr�fte nutzten, um die landwirtschaftliche Produktion von Gilneas w�hrend und nach der Zeit der Industrialisierung zu steigern.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/wc_religions.csv:155]: Capitalized two cases of Gilneas.
```suggestion
old_ways_DESC;The old ways are not widespread, but have existed on the fringes of the gilnean society. The religious leaders of what was in Gilneas referred to as the "old ways" eventually became "harvest-witches". Those who used their nature powers to augment Gilneas' agricultural output during and following its period of industrialisation.;Les anciennes m�thodes ne sont pas tr�s r�pandues, mais elles ont exist� en marge de la soci�t� giln�enne. Les chefs religieux de ce qu'on appelait � Giln�as les � anciennes m�thodes � ont fini par devenir des � sorciers des r�coltes �. Ce sont ceux qui ont utilis� leurs pouvoirs naturels pour augmenter la production agricole de Giln�as pendant et apr�s sa p�riode d'industrialisation.;Die alten Wege sind nicht weit verbreitet, existierten aber am Rande der gilneanischen Gesellschaft. Die religi�sen F�hrer dessen, was in Gilneas als die �alten Wege� bezeichnet wurde, wurden schlie�lich zu �Erntehexen�, die ihre Naturkr�fte nutzten, um die landwirtschaftliche Produktion von Gilneas w�hrend und nach der Zeit der Industrialisierung zu steigern.;;;;;;;;;;;x
```

File: localisation/wc_religions.csv [hunk 9]
```diff
@@ -177,30 +177,30 @@ GOD_ODYN;Odyn;Odyn;Odyn;;;;;;;;;;;x
...
...mother, have a much deeper connection with the water. Long have the tidesages listened to the sea to guide their ships and their people, gentle whispers imperceptible to most.;La M�re des mar�es est un �tre v�n�r� par de nombreux marins et par ceux dont la vie d�pend des vents marins changeants. La plupart des gens se tournent vers les mers pour se guider de la m�me fa�on que d'autres se tournent vers la Lumi�re, mais les eaugures, les pr�tres de la M�re des mar�es, ont un lien beaucoup plus profond avec l'eau. Depuis longtemps, les eaugures �coutent la mer pour guider leurs navires et leur peuple, des chuchotements doux imperceptibles pour la plupart.;Die Gezeitenmutter ist ein Wesen, das von vielen Seefahrern verehrt wird und von denen, deren Leben von ver�nderlichen Seewinden abh�ngt. Die meisten Menschen schauen auf das Meer, um sich leiten zu lassen, so wie andere auf das Licht schauen, aber die Gezeitenweisen, die Priester der Gezeitenmutter, haben eine viel tiefere Verbindung mit dem Wasser. Seit langem schon h�ren die Gezeitenweisen auf das Meer, um ihre Schiffe und ihre Menschen zu lenken, ein leises Fl�stern, das f�r die meisten Menschen nicht wahrnehmbar ist.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/wc_religions.csv:203]: Capitalized Light, added hearing to the last sentence.
```suggestion
tidemother_DESC;The Tidemother is a being worshipped by many sailors and those whose lives depend on mutable sea winds. Most people look to the seas for guidance the same way others look to the Light, but the tidesages, the priest of the Tidemother, have a much deeper connection with the water. Long have the tidesages listened to the sea to guide their ships and their people, hearing gentle whispers imperceptible to most.;La M�re des mar�es est un �tre v�n�r� par de nombreux marins et par ceux dont la vie d�pend des vents marins changeants. La plupart des gens se tournent vers les mers pour se guider de la m�me fa�on que d'autres se tournent vers la Lumi�re, mais les eaugures, les pr�tres de la M�re des mar�es, ont un lien beaucoup plus profond avec l'eau. Depuis longtemps, les eaugures �coutent la mer pour guider leurs navires et leur peuple, des chuchotements doux imperceptibles pour la plupart.;Die Gezeitenmutter ist ein Wesen, das von vielen Seefahrern verehrt wird und von denen, deren Leben von ver�nderlichen Seewinden abh�ngt. Die meisten Menschen schauen auf das Meer, um sich leiten zu lassen, so wie andere auf das Licht schauen, aber die Gezeitenweisen, die Priester der Gezeitenmutter, haben eine viel tiefere Verbindung mit dem Wasser. Seit langem schon h�ren die Gezeitenweisen auf das Meer, um ihre Schiffe und ihre Menschen zu lenken, ein leises Fl�stern, das f�r die meisten Menschen nicht wahrnehmbar ist.;;;;;;;;;;;x
```

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**